src/main/java/Figure.java

```diff
@@ -0,0 +1,9 @@
+public abstract class Figure {
+
+    private Color color;
```

Comment:
what's the point of this field if it's impossible to get it accessed?

Reply:
removed

Reply:
that makes sense to keep this variable here, cause it's used in several classes just add getter for it and constructor, so you would be able to initialize it

Reply:
understood
made Color in Figure and super in all figures